### ISSUE TITLE
Improve Documentation (Readme)

### ISSUE THREAD

**Issue by aakatev** [roles-ansible/role-scalelite#1]
I would suggest to improve the docs. Especially on how to get the role running. I am not an ansible developer, but would be interested to use this role. Also, stuff like infrastructure requirements (what specifically needs to be provisioned, even if it's a duplicate of the scalelite official specs) would be nice to have.

**Comment by DO1JLR** [comment]
You're right. I should really get to work on the docs. Right after I fixed the ruby part so that it is no longer deployed in development mode but in production mode.
